src/Api.php

```diff
@@ -151,6 +151,16 @@ public function call($method, $path, array $data = [], array $headers = []) {
     }
 
     try {
+      $auth = $resource->invokeAuthenticationService($this->getUser(), $request);
```

Comment:
I think this can't be here. It can be in restapi_page_callback(). Api() is called when the request is basically ready to go already, authentication needs to have been dealt with. This is awkward when things like a multi call happen (because then the authentication fires on each call?) or when internal calls are made.